backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5575110_sys_gh10412_AddSysconfig.sql

```diff
@@ -0,0 +1,5 @@
+-- 2020-12-16T13:50:13.970Z
+-- URL zum Konzept
+INSERT INTO AD_SysConfig (AD_Client_ID,AD_Org_ID,AD_SysConfig_ID,ConfigurationLevel,Created,CreatedBy,Description,EntityType,IsActive,Name,Updated,UpdatedBy,Value) VALUES (0,0,541351,'O',TO_TIMESTAMP('2020-12-16 10:50:11','YYYY-MM-DD HH24:MI:SS'),100,'If the current date - the value from the haddex date column of a product is bigger than the value from this sys config so that product can''t be added to a sales orderline. This value must be in months.','D','Y','MAX_HADDEX_AGE_IN_MONTHS',TO_TIMESTAMP('2020-12-16 10:50:11','YYYY-MM-DD HH24:MI:SS'),100,'24')
+;
```

Comment:
this sysconfig was already added by backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5574550_sys_gh10412HaddexColumnsToProductTableAndSysConfigsAndErrMsgAndProductWindowAdjustments.sql

That's why this script is failing.